pkg/ddc/alluxio/utils.go

```diff
@@ -243,7 +243,7 @@ func (e *AlluxioEngine) parseFuseImage(image string, tag string, imagePullPolicy
 }
 
 func (e *AlluxioEngine) GetMetadataInfoFile() string {
-	return cdatabackup.BACPUP_PATH_POD + "/" + e.GetMetadataInfoFileName()
+	return cdatabackup.ALLUXIO_BACPUP_PATH_POD + "/" + e.GetMetadataInfoFileName()
```

Comment:
typo here. Should be `ALLUXIO_BACKUP_PATH_POD` and `GOOSEFS_BACKUP_PATH_POD`

Reply:
tks, i will fix it.